packages/block-editor/src/components/block-caption/index.native.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { Caption, RichText } from '@wordpress/block-editor';
+import { compose } from '@wordpress/compose';
+import { withDispatch, withSelect } from '@wordpress/data';
+
+const BlockCaption = ( { accessible, accessibilityLabel, onBlur, onChange, onFocus, isSelected, shouldDisplay, text } ) => (
```

Comment:
Nitpick: Could we break this prop list onto multiple lines? It's getting pretty long for one line.

Reply:
I agree! I'll update it =)